src/common/pipe.c

```diff
@@ -14,10 +14,11 @@
 #include "acconfig.h"
 
 #include "common/pipe.h"
-#include "include/compat.h"
```

Comment:
i am not sure why we want to remove this. `VOID_TEMP_FAILURE_RETRY()` is defined in `compat.h`.